resources/versioning-policy.md

```diff
@@ -52,6 +52,12 @@ This kind of change will lead us to increase the MINOR to let the people know ab
 
 🚨 Any change about the relevancy that is related to API usage, and thus, that may impact users to change their code (for instance changing the default `matchingStrategy` value) is not related to this specific section and would lead us to increase the MAJOR.
 
+### New "variant" type addition
+
+We don't consider breaking to add a new type to an already existing list of variant. For example, adding a new type of `task`, or a new type of error `code`.
+
+We are aware some strongly typed language code bases could be impacted, and our recommendation is to handle the possibility of an unknown type when deserializing Meilisearch's response.
```

Comment:
What about this?
```suggestion
We are aware some strongly typed language code bases could be impacted, and our recommendation is to handle the possibility of having an unknown type when deserializing Meilisearch's response.
```

I got @irevoire's idea, and I agree with it. I think adding this verb will tell the user that they can expect an unknown type and they should handle it by themselves.